### PR TITLE
Cosmetics on slides for the linear model

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Half-way done. But @TwsThomas , maybe you can have a look.

### CODE REVIEW

File: slides/linear_model.md [hunk 1]
```diff
@@ -183,24 +185,32 @@ The axes correspond to x1, x2 and the color corresponds to the target label y.
...
+If we have too many parameters with regards to the number of samples, the
+linear model can overfit: it assigns non-zero weights to associations by
+change.
 
-With weights penalty, we include the value of the model's weights within the
-objective function (i.e. the sum of the squared residuals). So a penalized model
-should choose lower weights for almost a similar fit.
+The solution is to regularize the model: to foster less complex
+solutions. For this, the is slightly biased to
+should choose smaller weights for almost a similar fit.
```
Comment (TwsThomas) [inline slides/linear_model.md:209]:
```suggestion
solutions. For this, the regularized model is slightly biased to
choose smaller weights for almost a similar fit.
```

File: slides/linear_model.md [hunk 2]
```diff
@@ -183,24 +185,32 @@ The axes correspond to x1, x2 and the color corresponds to the target label y.
...
 
-The most classic way to limit its complexity is to push coefficients toward
-small values. Such model is called *Ridge*
 ???
 
-If we have too many parameters w.r.t. the number of samples, it is advised to
-penalize the parameters of our models.
+If we have too many parameters with regards to the number of samples, the
+linear model can overfit: it assigns non-zero weights to associations by
+change.
```
Comment (TwsThomas) [inline slides/linear_model.md:205]:
```suggestion
chance.
```

**Comment by GaelVaroquaux** [comment]
I worked a bit more on this, and it is ready for review. @TwsThomas , can you have a look!

Thanks!

File: slides/linear_model.md [hunk 3]
```diff
@@ -292,29 +315,30 @@ Multinomial logistic regression is a natural extension of logistic regression.
...
 
-* Linear model are good baselines for:
+* Good and understandable baselines for:
  - regression: linear regression + regularization = Ridge
  - classification: logistic regression + fine tune `C`
 
-* very fast to train
+* Very fast to train
+
 * Better when *nb of features* > *nb of samples*
```
[the remark this answers is not in the document]
Comment (GaelVaroquaux) [reply slides/linear_model.md:342]: No, it was correct: linear models linear models are better when n_features >> n_samples

[the remark this answers is not in the document]
Comment (GaelVaroquaux) [reply slides/linear_model.md:342]: No, we didn't talk about it in the slides, but I still think that it is important and that it is good to put it there.

**Pull request merged**